src/components/common/TmSelectNetwork.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <SessionFrame>
+    <div class="select-network">
+      <h2 class="session-title">Select a Network</h2>
+      <div class="select-networks-list">
+        <LiSession
+          v-for="network in sortedNetworks"
+          :key="network.title"
+          :img="network.id"
+          :title="network.title"
+          :route="{ name: 'create', params: { network: network.id } }"
+        />
+      </div>
+    </div>
+  </SessionFrame>
+</template>
+
+<script>
+import LiSession from "common/TmLiSession"
+import SessionFrame from "common/SessionFrame"
+import gql from "graphql-tag"
+
+export default {
+  name: `select-network`,
+  components: {
+    SessionFrame,
+    LiSession
+  },
+  data: () => ({
+    networks: []
+  }),
+  computed: {
+    sortedNetworks() {
```

Comment:
this logic is duplicated on the networks page. let's try to re-use.

Reply:
I am afraid this is quite difficult to do otherwise... The logic you are referring to is located in the PageNetworks component. But then, how can I use `@click=$router.push(`/create`)` on NetworkItem?

I hope I can explain well enough where the difficulty lies... if I just import PageNetworks in its totality, then how can I modify the inner components PageNetworks brings within? Otherwise the user can just click anywhere in the component (without selecting any network) and then make it to create an account.

Reply:
i don't know what you mean about the router - but it's fine. `LiNetwork` should be used for sure.

Reply:
I was meaning the `@click=$router.push`directive :wink:

Reply:
`NetworkList` could actually be used and refactored so it takes an ´onClick` handler. `PageNetworks` then implements the `onClick` handler

Reply:
Done! :muscle: 

Now it looks like this (guess we need to tweak the styles a bit):

<img width="540px" src="https://user-images.githubusercontent.com/40721795/72893789-3cfed880-3d1a-11ea-92f0-dc82b7e1112a.png">